### PR TITLE
[sailfish-webview] Add handling downloadpicker to PickerOpener. Contributes to JB#53798

### PR DESCRIPTION
**Pull request by atatarov**

### CODE REVIEW

**Review by llewelld — COMMENTED** [review]
Just a couple of small points. In practice this is working nicely.

**Review by llewelld — APPROVED** [review]
Looks good; thanks.

**Comment by llewelld** [comment]
These three are now ready to merge as far as I'm concerned, as soon as they have OMP approval too.
https://github.com/sailfishos/sailfish-components-webview/pull/97 (this PR)
https://git.sailfishos.org/mer-core/embedlite-components/merge_requests/118
https://github.com/sailfishos/sailfish-browser/pull/821

**Review by okodron — APPROVED** [review]

**Pull request merged**
